mmdet/models/backbones/resnet.py

```diff
@@ -46,9 +46,9 @@ def __init__(self,
             planes,
             planes,
             3,
-            stride=stride,
-            padding=dilation,
-            dilation=dilation,
+            stride=1,
+            padding=1,
```

Comment:
`padding` should be 0 instead. Actually you can just remove padding and dilation to use default values.

Reply:
the kernel size of conv2 is 3, so i suppose the padding should be 1.

Reply:
Oh I mistook the kernel size. It should be fine.

> the kernel size of conv2 is 3, so i suppose the padding should be 1.